examples/experimental/dagster-airlift/dagster_airlift/in_airflow/base_asset_operator.py

```diff
@@ -141,40 +139,39 @@ def launch_runs_for_task(self, context: Context, dag_id: str, task_id: str) -> N
         ]
         if not filtered_asset_nodes:
             raise Exception(f"No asset nodes found to trigger for task {dag_id}.{task_id}")
-        for asset_node in filtered_asset_nodes:
-            assets_to_trigger_per_job[_build_dagster_job_identifier(asset_node)].append(
-                asset_node["assetKey"]["path"]
+        if (
+            not len(
+                {_get_implicit_job_identifier(asset_node) for asset_node in filtered_asset_nodes}
             )
-        logger.info(f"Found {len(assets_to_trigger_per_job)} jobs to trigger")
-
-        triggered_runs = []
-        for job_identifier, asset_key_paths in assets_to_trigger_per_job.items():
-            logger.info(f"Triggering run for {job_identifier} with assets {asset_key_paths}")
-            run_id = self.launch_dagster_run(
-                context,
-                session,
-                dagster_url,
-                _build_dagster_run_execution_params(
-                    self.default_dagster_run_tags(context),
-                    job_identifier,
-                    asset_key_paths,
-                ),
+            == 1
+        ):
+            raise Exception(
```

Comment:
what's the behavior for named jobs?

Reply:
In 1.8 and above we _always_ create an implicit asset job, so this should work even with a named job

Reply:
Ah